setup.py

```diff
@@ -6,7 +6,7 @@
 REQUIREMENTS = [
     "configsuite>=0.6",
     "cwrap>=1.6",
-    "ecl2df>=0.5",
+    "ecl2df~=0.8.1",
```

Comment:
Have me made an issue of why this needs to be pinned?

Reply:
We don't really need to actually... I could change it to `ecl2df>=0.8.1`